docs/index.html

```diff
@@ -1353,6 +1353,27 @@ <h4 class="headline-3">Module: Button</h4>
 		</div>
 	</section>
 
+	<section class="grid">
+		<div class="grid-row">
+			<div class="grid-cell x3 hint-text">
+				<h4 class="headline-3">Module: Issue Labels</h4>
+				<p>
+					This Module helps you to set filter labels. The ID <code>filter-issue-status</code> wraps the labels. The <code>filter-label</code> class makes sure, that all labels are wrapped in a container with a 100% width and the text is centered. The current used label gets the class <code>active</code> as well to show in wich state the ticket is currently. 
+				</p>
+				<p>There are 6 labels you can use:</p>
```

Comment:
this and the next paragraph can be removed as we have the context above. 
(first context, then code explanation).